models/cex/ethereum/cex_ethereum_flows.sql

```diff
@@ -0,0 +1,36 @@
+{{config(
+        schema = 'cex_ethereum',
+        alias = 'flows',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['flow_type', 'block_number', 'tx_index', 'unique_key']
```

Comment:
is just `unique_key` not enough? if it's unique in transfers, then you just join to `cex_ethereum.addresses` to obtain a name, it should remain unique?

Reply:
changed to ['flow_type', 'unique_key'] since one exchange's outflow could be another's inflow, and flow_type is also needed for transfers between diff wallets of same cex.